src/pages/ItemList.js

```diff
@@ -140,9 +150,66 @@ function ItemList({ token }) {
           <button onClick={() => setSearchInput(() => '')}>Reset</button>
           {list.map((doc) => {
             const wasCheckedInLast24Hours =
+              // now - doc.last_purchased_date < ONE_SECOND;
               now - doc.last_purchased_date < ONE_DAY;
+
+            const inactiveItem =
+              msToDay(now - doc.last_purchased_date) >=
+              doc.previous_estimate * 2;
+
+            const itemStatus = () => {
+              //assigns style based on purchase urgency
+              //items are grey if purchased recently because of small estimate
+              //45 days from now for testing
+              console.log(msToDay(now - 1648431178591)); //45 days ago 5/12/22**
+              if (
+                doc.last_purchased_date !== null &&
+                doc.previous_estimate !== 0 &&
+                inactiveItem
+              ) {
+                console.log('inactive');
+                return {
+                  color: 'grey',
+                  label: 'inactive item',
+                };
+              } else if (doc.previous_estimate >= 30) {
+                console.log('>= 30');
+                return {
+                  color: 'red',
+                  label: `${doc.previous_estimate} days expected until purchase needed`,
+                };
+              } else if (
+                doc.previous_estimate > 7 &&
+                doc.previous_estimate < 30
+              ) {
+                return {
+                  color: 'yellow',
+                  label: `${doc.previous_estimate} days expected until purchase needed`,
+                };
+              } else if (doc.previous_estimate <= 7) {
+                console.log(7);
+                return {
+                  color: 'green',
+                  label: `${doc.previous_estimate} days expected until purchase needed`,
+                };
+              } else {
+                console.log('nothing');
```

Comment:
Is the return object for the else block ever used?

Reply:
Nope, it was used when we were working out the logic, but I will remove it now. Thanks for catching that!